services/brig/src/Brig/API/Client.hs

```diff
@@ -404,7 +409,8 @@ noPrekeys u c = do
     field "user" (toByteString u)
       ~~ field "client" (toByteString c)
       ~~ msg (val "No prekey found. Ensuring client does not exist.")
-  Intra.rmClient u c
+  -- Intra.rmClient u c
```

Comment:
I assume at some point we need to actually delete the clients correctly everywhere or is that not the case? Also would that require some script to prune the required Casandras?